### PR TITLE
fix: Cmd+Tab not working when exiting kiosk mode

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Closes https://github.com/electron/electron/issues/33503.
Refs https://github.com/electron/electron/pull/25470.

Fixes an issue on macOS when trying to switch processes with `Cmd+Tab` after exiting Kiosk Mode. We need to ensure that presentation options are reset _after_ exiting fullscreen, otherwise the fullscreen action takes precedence and causes them to improperly be reset.

Tested with https://gist.github.com/40902920c8fb6c0e2ebcd8fdb936cbf6.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: Fixes an issue on macOS when trying to switch processes with `Cmd+Tab` after exiting Kiosk Mode.

### CODE REVIEW

File: shell/browser/native_window_mac.mm [hunk 1]
```diff
@@ -1022,17 +1022,20 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
         NSApplicationPresentationHideDock |
         NSApplicationPresentationHideMenuBar |
         NSApplicationPresentationDisableAppleMenu |
-        NSApplicationPresentationDisableProcessSwitching |
+        // NSApplicationPresentationDisableProcessSwitching |
```
Comment (zcbenz) [inline shell/browser/native_window_mac.mm:1025]: Is this change still needed?

Comment (codebytere) [reply shell/browser/native_window_mac.mm:1025]: oops, extracting

**Review by zcbenz — APPROVED** [review]
